install_config/master_node_configuration.adoc

```diff
@@ -1013,3 +1013,213 @@ To launch a node server using a node configuration file:
 ----
 $ openshift start node --config=/openshift.local.config/node-<node_hostname>/node-config.yaml
 ----
+
+[[config-logging-levels]]
+== Configuring Logging Levels
+
+{product-title} uses the `systemd-journald.service` to collect log messages for debugging, using five log message severities. The logging levels are based on Kubernetes logging conventions, as follows:
+
+.Log Level Options
+[cols="3a,6a",options="header"]
+|===
+
+|Option |Description
+
+|0|Errors and warnings only
+|2|Normal information
+|4|Debugging-level information 
+|6|API-level debugging information (request / response)
+|8|Body-level API debugging information
+|===
+
+You can control which INFO messages are logged by setting the loglevel option in the in `/etc/sysconfig/atomic-openshift-node` or `/etc/sysconfig/atomic-openshift-master` file. Configuring the logs to collect all messages can lead to large logs that are difficult to interpret and can take up excessive space. Collecting all messages should only be used in debug situations.
```

Comment:
File name markup is:

```
*_filename_*
```

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference